src/rviz/render_panel.h

```diff
@@ -100,6 +101,9 @@ Q_OBJECT
 
   virtual void sceneManagerDestroyed( Ogre::SceneManager* source );
 
+Q_SIGNALS:
+  void onViewportMouseEvent( const ViewportMouseEvent& event);
```

Comment:
in Qt signals don't start with "on", that is reserved for slots. ie signal mouseEvent -> slot: onMouseEvent